pyvista/core/grid.py

```diff
@@ -110,11 +110,11 @@ class RectilinearGrid(_vtk.vtkRectilinearGrid, Grid):
     >>> vtkgrid = vtk.vtkRectilinearGrid()
     >>> grid = pyvista.RectilinearGrid(vtkgrid)
 
-    Create from NumPy arrays
+    Create from NumPy arrays.
 
     >>> xrng = np.arange(-10, 10, 2)
     >>> yrng = np.arange(-10, 10, 5)
-    >>> zrng = np.arange(-10, 10, 1)
+    >>> zrng = np.arange(-10, 10, 1.1)
```

Comment:
Why was this changed to 1.1?

Reply:
```suggestion
    >>> zrng = np.arange(-10, 10, 1)
```

Reply:
I was playing around with something in testing and forgot to remove it from the change. Sharp eyes as always @adeak.